core/Updates/2.11.1-b3.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Updates;
+
+use Piwik\Config;
+use Piwik\Development;
+use Piwik\Updates;
+
+class Updates_2_11_1_b3 extends Updates
+{
+    /**
+     * Here you can define any action that should be performed during the update. For instance executing SQL statements,
+     * renaming config entries, updating files, etc.
+     */
+    static function update()
+    {
+        if (!Development::isEnabled()) {
+            return;
+        }
+
+        $config  = Config::getInstance();
+        $dbTests = $config->database_tests;
+
+        if ($dbTests['username'] === '@USERNAME@') {
+            $dbTests['username'] = 'root';
```

Comment:
Basically, I just wanted to have it as before. Before, in global config was the default username `root` which I changed to `@USERNAME@`. By setting it back to root for developers who update I make sure it is the same as before. In theory, developers could use a different username for there regular database in `$dbTests['username']`. I used to have this in the `piwik-dev-environment` where I had one user `piwik` for the normal database that Piwik uses and one user `root` for tests.